examples/example_launch_training.py

```diff
@@ -83,6 +88,20 @@ def create_parser():
         "output_prediction_horizon": args.output_prediction_horizon,
     }
 
+    input_robot_data_spec: RobotDataSpec = {
+        "robot_id": {
+            # nc.DataType.JOINT_POSITIONS: [],
+            # nc.DataType.JOINT_VELOCITIES: [],
+            nc.DataType.RGB_IMAGES: ["angle"],
+        }
+    }
+
+    output_robot_data_spec: RobotDataSpec = {
+        "robot_id": {
+            nc.DataType.JOINT_POSITIONS: ACTION_KEYS,
```

Comment:
I believe action keys actually has gripper data in it. 
See https://github.com/NeuracoreAI/neuracore/blob/main/examples/example_local_endpoint.py#L22